relay/kusama/src/lib.rs

```diff
@@ -1919,11 +1922,11 @@ impl Runtime {
 		};
 
 		// We assume un-delayed 6h eras.
-		let era_duration = 6 * HOURS;
+		let era_duration = 6 * 60 * 60 * 1000;
```

Comment:
Why was this changed? And is the comment now incorrect?

Reply:
unified them in a better way in [a5d3ed3](https://github.com/polkadot-fellows/runtimes/pull/443/commits/a5d3ed3cbd0ab5a5f93e44242805ecc4e69b3dd2)